.github/workflows/code-quality.yml

```diff
@@ -29,7 +29,17 @@ jobs:
       - name: Cargo fmt
         run: cargo fmt --check
       - name: Cargo clippy
-        run: cargo clippy --all-targets --all-features --
-          -D warnings
+        run: cargo clippy --all-targets --all-features -- -D warnings
       - name: Prettier for JS
         run: npx prettier "{lib,examples,test}/**/*.js" --check
+      - name: Prepare index file verification
+        run: |
+          mv index.d.ts old_index.d.ts
+          mv index.js old_index.js
+      - name: Build
+        run: npm run build
+      - name: Check if got diffrent results
+        run: |
+          diff index.d.ts old_index.d.ts
+          diff index.js old_index.js
+      
```

Comment:
Could you please cherry-pick this in another branch and open a separate PR for this? I'd like to have this merged ASAP, to immediately gain this check for all other PRs.

Reply:
I'll try to do it before our meeting